libs/ts-rest/next/src/lib/ts-rest-next.ts

```diff
@@ -324,16 +324,20 @@ const handlerFactory = (
         }
 
         if (!pathParamsResult.success) {
-          return res.status(400).json(pathParamsResult.error);
+          res.status(400).json(pathParamsResult.error);
+          return;
```

Comment:
What does removing the returns from the next lib accomplish, sorry? I understand the rationale in the express project but less so here, consistency?